OptionParser.hpp

```diff
@@ -21,7 +21,7 @@ class OptionParser {
         if (argc > 1) {
             for (int i=1; i<argc; ++i) {
                 std::string temp = argv[i];
```

Comment:
What if i add a new argument "---123" and write the following test case?
`CHECK( parser.GetParam(6) == "---123" );`